__init__.py

```diff
@@ -191,7 +192,10 @@ def _start_new_timer(self, message):
         else:
             timer = self._build_timer(duration, name)
             self.active_timers.append(timer)
-            self.active_timers.sort(key=lambda x: x.expiration)
+            self.active_timers.sort(key=lambda tmr: tmr.expiration)
+            if len(self.active_timers) == 1:
+                self._show_gui()
+                self._start_display_update()
             self._speak_new_timer(timer)
             self._save_timers()
 
```

Comment:
I don't understand why it would be.  The intent of this code is to show the GUI page when the first timer is set.  The assumption being that the page will continue to display until no more timers are active.